src/main/java/seedu/intrack/logic/commands/FindPCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.intrack.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.intrack.commons.core.Messages;
+import seedu.intrack.model.Model;
+import seedu.intrack.model.internship.PositionContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all internships in the internship tracker whose position contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindPCommand extends Command {
```

Comment:
```suggestion
public class FindPositionCommand extends Command {
```
Would it be better to rename it to FindPositionCommand for a clearer description on what the command does?